client/src/boot/applyTransforms.js

```diff
@@ -31,6 +32,15 @@ const togglePristineState = (field, isPristine = false) => {
 };
 
 const applyTransforms = () => {
+  Injector.transform(
+    'field-holders',
+    (updater) => {
+      const fields = [
+        'FieldGroup',
+      ];
+      fields.forEach((field) => updater.component('FieldGroup', fieldHolder, `${field}Holder`));
```

Comment:
This feels like a premature optimisation - was there a justification for making this an array off the bat?

Reply:
agreed, add a few more fields to it to justify :)

Reply:
Hmm. Originally this body of work included _all_ affected form fields, however that required far more than a simple pull request as a proof of concept (such as complete regression testing of _all_ components in the CMS). I worked on this along side @unclecheese and had (foolishly) omitted context from the PR thinking it was pre-gained knowledge (plus was in a rush at the time). That's bad for open source, allow me to rectify in the opening comment.

It _is_ premature optimisation... but it is with the intent to revisit in the near future to complete a much more overarching (and completely separate - c.f. #510) story. This is the minimal to get feedback on the approach, and complete the task at hand (https://github.com/dnadesign/silverstripe-elemental/issues/181)

Reply:
add a `// @todo: we need more!` comment then, I'm fine with this